src/libraries/System.Net.Quic/src/Interop/MsQuicEnums.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net.Quic.Implementations.MsQuic.Internal
```

Comment:
Is there any specific order you chose for these enums in here? If not, I think it'd be best to match the order in msquic.h so that it's easier to do a line by line scan to make sure everything matches.